src/inputs/input.css

```diff
@@ -0,0 +1,174 @@
+div.omrs-input-group {
+  margin-bottom: 1.5rem;
+  position: relative;
+  width: 20.4375rem;
+}
+
+input.omrs-input-underlined,
+input.omrs-input-filled,
+input.omrs-input-outlined {
+  font-size: 1rem;
+  display: block;
+  box-sizing: border-box;
+  width: 100%;
+  height: 3.5rem;
+  padding-left: 0.875rem;
+  outline: none;
+}
+
+input.omrs-input-underlined,
+input.omrs-input-filled {
+  border: none;
+  border-bottom: 2px solid var(--omrs-color-ink-lowest-contrast);
+  border-radius: 0.25rem 0.25rem 0px 0px;
+  vertical-align: middle;
+  line-height: 147.6%;
+}
+
+input.omrs-input-outlined {
+  background: rgba(255, 255, 255, 0.01);
+  border: 1px solid var(--omrs-color-ink-lowest-contrast);
+  border-radius: 0.25rem;
+}
+
+input.omrs-input-filled {
+  background: var(--omrs-color-bg-low-contrast);
+}
+
+input.omrs-input-underlined {
+  background: var(--omrs-color-ink-white);
+}
+
+input.omrs-input-underlined ~ label,
```

Comment:
By using the `~` css operator, we're applying it to *all labels that are siblings*, not just one label.

The way you've set up the HTML contract though, I think maybe that's okay?? We don't really need to support the `input` having multiple `<label>` elements as siblings?

Reply:
I don't think we would need an input to have multiple labels. I haven't seen that scenario before and it's not part of the style guide. Do you think we might need that in future?

Reply:
Nah I don't think so. Let's stick with this.